shotover-proxy/examples/cassandra-passthrough/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   cassandra-one:
     image: library/cassandra:3.11.10
     ports:
-      - "9043:9042"
```

Comment:
Can you move this into a separate PR?
It might be the correct approach but it will need some consideration and we will want to do the same for redis.